src/Components/OATGraphViewer/Internal/GraphViewerControls/GraphViewerControls.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Controls, ControlButton } from 'react-flow-renderer';
+import {
+    classNamesFunction,
+    styled,
+    FocusZone,
+    Icon,
+    Stack
+} from '@fluentui/react';
+import { CONTROLS_Z_INDEX } from '../../../../Models/Constants/OatStyleConstants';
+import { OatGraphContextActionType } from '../../../../Models/Context/OatGraphContext/OatGraphContext.types';
+import HeaderControlButton from '../../../HeaderControlButton/HeaderControlButton';
+import HeaderControlGroup from '../../../HeaderControlGroup/HeaderControlGroup';
+import {
+    IGraphViewerControlsProps,
+    IGraphViewerControlsStyleProps,
+    IGraphViewerControlsStyles
+} from './GraphViewerControls.types';
+import { getStyles } from './GraphViewerControls.styles';
+import { useOatGraphContext } from '../../../../Models/Context/OatGraphContext/OatGraphContext';
+import { useExtendedTheme } from '../../../../Models/Hooks/useExtendedTheme';
+
+const getClassNames = classNamesFunction<
+    IGraphViewerControlsStyleProps,
+    IGraphViewerControlsStyles
+>();
+
+const GraphViewerControls: React.FC<IGraphViewerControlsProps> = (props) => {
+    const {
+        legendButtonId,
+        miniMapButtonId,
+        modelListButtonId,
+        onApplyAutoLayoutClick,
+        styles
+    } = props;
+
+    // contexts
+    const { oatGraphDispatch, oatGraphState } = useOatGraphContext();
+
+    // state
```

Comment:
Are these placeholders for future dev or should they be removed?

Reply:
ehh.. these are just part of the generated template. I can remove them though.